config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatatransferconfigs.bigquerydatatransfer.cnrm.cloud.google.com.yaml

```diff
@@ -668,10 +668,9 @@ spec:
                     type: string
                 type: object
               scheduleOptionsV2:
-                description: V2 options customizing different types of data transfer
-                  schedule. This field supports existing time-based and manual transfer
-                  schedule. Also supports Event-Driven transfer schedule. ScheduleOptionsV2
-                  cannot be used together with ScheduleOptions/Schedule.
+                description: Options customizing different types of data transfer
```

Comment:
I previously added the `scheduleOptionsV2` field manually using the field comments from [1]. However, the correct field comments is [2]. Only a slight difference between the two.

[1] https://github.com/googleapis/googleapis/blob/f4b02617cbc3db919fa05c5ad782ef211dd90889/google/cloud/bigquery/datatransfer/v1/transfer.proto#L101-L104

[2] https://github.com/googleapis/googleapis/blob/f4b02617cbc3db919fa05c5ad782ef211dd90889/google/cloud/bigquery/datatransfer/v1/transfer.proto#L231-L234